LM-1201/99-beta/hello-python.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""DevNet Express - Hello Python!
+
+Running this script will help you verify that your Python environment is setup
+correctly.  It will display information about your Python environment including
+the location of your Python interpreter, where your `pip install ...` packages
+are being installed and more.
+
+Usage:
+    python hello-python.py
+
+"""
+
+
+__author__ = "Chris Lunsford"
+__author_email__ = "chrlunsf@cisco.com"
+__copyright__ = "Copyright (c) 2018 Cisco and/or its affiliates."
+__license__ = "Cisco Sample Code License, Version 1.0"
```

Comment:
Hi again Chris, we need to make sure this Cisco Sample Coe License v 1.0 gets into a LICENSE file in this repo, right?